Packs/CyberArkPAS/Integrations/CyberArkPAS/CyberArkPAS.yml

```diff
@@ -0,0 +1,913 @@
+  name: max_fetch
+  required: false
+  type: 0
+- defaultvalue: 7 days
+  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days, 3
+    months, 1 year)
+  name: fetch_time
+  required: false
+  type: 0
+- defaultvalue: 0
+  display: Query, CyberArk Pas value (0.0-100.0)
+  name: query
+  required: false
+  type: 0
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: This is the Hello World integration for getting started.
+display: CyberArk PAS
+name: CyberArkPAS
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: The name of the user.
+      isArray: false
+      name: username
+      required: true
+      secret: false
+    - default: false
+      description: User type according to the license.
+      isArray: false
+      name: user_type
+      required: false
+      secret: false
+    - default: false
+      description: The CyberArkPAS interfaces that this user is not authorized to use.
+        e.g. - "PSM", "PSMP"
+      isArray: true
+      name: non_authorized_interfaces
+      required: false
+      secret: false
+    - default: false
+      description: The location in the Vault where the user will be created. Must
+        begin with "\\".
+      isArray: false
+      name: location
+      required: false
+      secret: false
+    - default: false
+      description: The date when the user expires.
+      isArray: false
+      name: expiry_date
+      required: false
+      secret: false
+    - default: false
+      description: The password that the user will use to log on for the first time.
+      isArray: false
+      name: password
+      required: true
+      secret: true
+    - auto: PREDEFINED
+      default: false
+      defaultValue: 'true'
+      description: Whether or not the user must change their password from the second
+        log on onward.
+      isArray: false
+      name: change_password_on_the_next_logon
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: 'false'
+      description: Whether the user’s password will not expire unless they decide
+        to change it.
+      isArray: false
+      name: password_never_expires
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: |-
+        The user permissions.
+        Valid values:
+        AddSafes
+        AuditUsers
+        AddUpdateUsers
+        ResetUsersPasswords
+        ActivateUsers
+        AddNetworkAreas
+        ManageDirectoryMapping
+        ManageServerFileCategories
+        BackupAllSafes
+        RestoreAllSafes
+      isArray: true
+      name: vault_authorization
```

Comment:
that's a tricky one. if the options are fixed, why not use the list option?
if you an pass more than one - maybe add an e.g?

Reply:
you can pass more than one, I'll add e.g